### DISCUSSION TITLE
Ability to set mouse button modifier key using the keys registered via steer mouse

### DISCUSSION THREAD

**Discussion by smartymcfly** [rxhanson/RectanglePro-Community#557] [Ideas]
Ability to set mouse button modifier key using the keys registered via steer mouse because a single button press is being cancelled out by the app and also the button press is cancelling the config of the buttons in steer mouse.

Can this be done now, or if not can it please be added? I want to be able to use the shift button plus another button on the mouse to trigger throws without interrupting the other shortcuts setup via steer mouse.

Or if there is a way to set the button in steer mouse via rectangles:// app address that would also work.

![CleanShot 2024-09-03 at 18 10 28](https://github.com/user-attachments/assets/10dbaa03-a082-44cc-a220-25504db938ec)

**Comment by rxhanson** [comment]
Thanks for the feedback. I typically don't build out functionality that relies on other 3rd party apps, but I'll think about how this could go with just an implementation in Rectangle Pro.

**Reply by smartymcfly** [reply]
Thank You, it doesn't need to be with specific app there are a handful that all do something similar.  My github email is not the email I purchased rectangle pro with.
